### PR TITLE
feat: Copy resources to build output

### LINKED ISSUE #74: IntelliJ plugin should copy resources on save
In case of resource files they are not copied to target and not refreshed in application.

### PR DESCRIPTION
**Pull request by MarcinVaadin**
Fixes #74

### CODE REVIEW

**Comment by MarcinVaadin** [comment]
It should be done after compiling, not writing, otherwise output directory might not event exist.

File: src/main/kotlin/com/vaadin/plugin/copilot/handler/WriteBase64FileHandler.kt [hunk 1]
```diff
@@ -8,12 +8,12 @@ import com.intellij.psi.PsiFile
 import com.intellij.psi.PsiManager
 import java.io.File
 import java.nio.file.Files
-import java.util.Base64
+import java.util.*
```
Comment (Artur-) [inline src/main/kotlin/com/vaadin/plugin/copilot/handler/WriteBase64FileHandler.kt:11]: Let’s not use * here either

**Comment by Artur-** [comment]
It looks like this implements manual copying - should it not be the same as ”compile” for Java files, so that the save action handles it also when saving in the IDE?

File: src/main/kotlin/com/vaadin/plugin/copilot/handler/WriteBase64FileHandler.kt (hunk 1 above)
Comment (MarcinVaadin) [reply src/main/kotlin/com/vaadin/plugin/copilot/handler/WriteBase64FileHandler.kt:11]: Yes, I'm reverting, IntelliJ is changing it automatically

**Comment by MarcinVaadin** [comment]
> It looks like this implements manual copying - should it not be the same as ”compile” for Java files, so that the save action handles it also when saving in the IDE?

Let me have a second look if compile on save handles it also...

**Comment by Artur-** [comment]
If we are handling Java files both manually in the Copilot command and through the save action, then we should do the same for resources. But we need to let IntelliJ do the actual copying/compiling

**Comment by MarcinVaadin** [comment]
Will be continued in new, fresh PR

**Pull request closed without merging**